spec/support/capybara.rb

```diff
@@ -1,9 +1,24 @@
 require 'capybara/rspec'
 require 'capybara-screenshot/rspec'
-require 'capybara/poltergeist'
 require 'rack_session_access/capybara'
+require "selenium/webdriver"
+
+Capybara.register_driver :chrome do |app|
```

Comment:
I'm curious about why we are registering chrome and headless chrome? Is there somewhere we use the chrome driver, or somewhere we can flip it on for feature tests?

Reply:
Good point. I copied that from online sources, but we can probably remove the `:chrome` driver. I will remove and see if the tests still pass.